docs/_articles/en/user-guide/source-diff.md

```diff
@@ -27,6 +27,7 @@ Source Diff currently supports the following metadata:
 - Aura Event
 - Aura Interface
 - Aura Token
+- Custom Labels
```

Comment:
Very timely PR since I just noticed we forgot to update this. We should also add Layouts and Permission sets to it.

Reply:
oooo and this is a very timely comment @lcampos  because I'm working on a blog post.  I will add the Layouts & permission sets to the post.  (I had the custom labels, which is why I noticed the missing doc :) )

Reply:
Updated the supported metadata list with Layout and Permission Sets.